### ISSUE TITLE
when ever i change something in the rem they will appear twice in the queue

### ISSUE THREAD

**Issue by drstrangez0** [remnoteio/remnote-issues#256]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mfranzs** [comment]
Thanks for reaching out. Can you clarify what you mean? Are you changing the Rem within the queue?

**Comment by drstrangez0** [comment]
If I already practiced the rem and then add a new information to it in the editor it will come twice in the queue

**Comment by tfiwm** [comment]
Here is one such exhibiting the same behavior. I answered this card 8 times in three days! I started noticing this only after last update. Only few rems get into this state, don't know what triggers this

![image](https://user-images.githubusercontent.com/24322880/98459223-a20ea800-214d-11eb-9d5a-267b043d2fef.png)

**Comment by drstrangez0** [comment]
> Here is one such exhibiting the same behavior. I answered this card 8 times in three days! I started noticing this only after last update. Only few rems get into this state, don't know what triggers this
> 
> ![image](https://user-images.githubusercontent.com/24322880/98459223-a20ea800-214d-11eb-9d5a-267b043d2fef.png)

i think what triggers that is whenever you change anything in the rem so to avoid this just delete it and create a new flashcard

**Comment by Mesh959** [comment]
This is happening a lot. I kept seeing some rems appear twice in the queue on the same day. At first I thought it was a bug in which it wasn't saving my answers. But recently I noticed that they have different scheduling times.
These rems are appearing twice and are behaving like two separate cards

Below is one example - same card appearing twice in a queue with different scheduling times
![image](https://user-images.githubusercontent.com/74339445/98895716-5b160f00-24cd-11eb-9d0c-e53277081135.png)

![image](https://user-images.githubusercontent.com/74339445/98896027-19399880-24ce-11eb-8dc7-a55ef1691463.png)
